### PR TITLE
Add pip with python3 and pypy support

### PR DESCRIPTION
**Pull request by yorinasub17**
Adds two more managers:
- `pip3` for python3 pip
- `pip_pypy` for pypy pip

Test with `python setup.py test` or `python -m unittest tests.test_pip` for a shorter test cycle.

FYI @woohp

### CODE REVIEW

File: cappa.py [hunk 1]
```diff
@@ -69,9 +71,18 @@ def _assert_manager_exists(self, manager_type):
                 raise MissingExecutable('tsd not found')
             elif manager_type == 'pip':
                 raise MissingExecutable('pip not found')
+            elif manager_type == 'pip3':
```
Comment (ChristopherSchuster) [inline cappa.py]: lines 66-79 almost collapse to
```
raise MissingExecutable('{} not found'.format(manager_type)
```

**Comment by ChristopherSchuster** [comment]
LGTM. Merge with or without addressing comment.

**Comment by yorinasub17** [comment]
Addressed! Will merge once tests pass and then I will tag a new version.

**Comment by yorinasub17** [comment]
Test passed. Merging away!

**Pull request merged**